### DISCUSSION TITLE
Copy of item(s)

### DISCUSSION THREAD

**Discussion by cmllr** [directus/directus#7626] [Feature Requests]
Currently, Directus has some context features if a element is selected, e. g. to delete item(s) within the list view.

It would be great to have an option to create duplicates of the currently selected items.

**Comment by benhaynes** [comment]
Great idea. I think this would work well, we just need to catch unique field values (same on the Item Detail's "Save as Copy") to show a clear notification.

**Reply by cmllr** [reply]
I agree! Another idea: Not copy and save the item instantly and just open the new item form "preinitialized" with the data from the "copied" item. Might have the disadvantage of the constraint that only a single item can be copied. I don't know if it's directly better than my initial thought, but I just wanted to add this aswell!

**Reply by rijkvanzanten** [reply]
@cmllr That's already (sort of) possible by opening the item you want to copy, and then using the "Save as Copy" option under the save context menu 🙂

**Comment by rijkvanzanten** [comment]
We could re-use the same field duplication configuration as for save-as-copy, which could mean that it's up to the user to make sure unique field constraints aren't duplicated

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
